public/locales/pt/translation.json

```diff
@@ -32,6 +33,6 @@
   "loader": "A carregar a lista de ficheiros...",
   "downloadFiles": {
     "downloadAll": "Descarregar todos",
-    "downloading": "A arquivar..."
+    "downloading": "A descarregar..."
```

Comment:
@fsdiogo just to check, did these pt language changes get pulled from transifex, or did you edit them in the file manually? Do they need to be pushed to transifex?

Reply:
This project isn't on transifex yet; I was waiting to get some sort of "final copy" first because it's possible that many of these tags change.